common/source/docs/common-auxiliary-functions.rst

```diff
@@ -0,0 +1,633 @@
++----------------------+----------------------------+----------+---------+---------+
+|        16            | AUTO mode                  |    X     |    X    |    X    |
++----------------------+----------------------------+----------+---------+---------+
+|        17            | AUTOTUNE mode              |    X     |         |         |
++----------------------+----------------------------+----------+---------+---------+
+|        18            | LAND Mode                  |    X     |         |         |
++----------------------+----------------------------+----------+---------+---------+
+|        19            | Gripper                    |    X     |         |         |
++----------------------+----------------------------+----------+---------+---------+
+|        21            | Parachute Enable           |    X     |         |         |
++----------------------+----------------------------+----------+---------+---------+
+|        22            | Parachute Release          |    X     |         |         |
++----------------------+----------------------------+----------+---------+---------+
+|        23            | Parachute 3 Pos Switch     |    X     |         |         |
++----------------------+----------------------------+----------+---------+---------+
+|        24            | Reset Auto Mission to start|    X     |    X    |    X    |
++----------------------+----------------------------+----------+---------+---------+
+|        25            | Attitude Controller FF     |    X     |         |         |
++----------------------+----------------------------+----------+---------+---------+
+|        26            | Attitude Controller AccLim |    X     |         |         |
++----------------------+----------------------------+----------+---------+---------+
+|        27            | Retract Mount              |    X     |         |         |
++----------------------+----------------------------+----------+---------+---------+
+|        28            | Relay 1 On/off             |    X     |    X    |    X    |
++----------------------+----------------------------+----------+---------+---------+
+|        29            | Landing Gear               |    X     |    X    |         |
++----------------------+----------------------------+----------+---------+---------+
+|        30            | Lost Vehicle Sound         |    X     |    X    |    X    |
++----------------------+----------------------------+----------+---------+---------+
+|        31            | Motor Emergency Stop       |    X     |    X    |    X    |
++----------------------+----------------------------+----------+---------+---------+
+|        32            | Motor Interlock            |    X     |         |         |
++----------------------+----------------------------+----------+---------+---------+
+|        33            | Brake                      |    X     |         |         |
```

Comment:
"Brake" should probably become "BRAKE mode"

Reply:
I have been seeing and using modes in all caps, but we can change going forward...personally i think it differentiates the word as a mode...params are all caps, but almost always have an underscored form...so I think confusion is low probability....
This is indeed a flight mode, but I need to change the mode set wording below a bit since this is not an edge triggered mode switch like most are...

Reply:
Is it not "edge triggered"?  I think it should be just like other flight modes in many respects..

Reply:
that not what the existing descritption says on the CH7/8 optin switches in the wiki....its in brake on high, out of brake when low.....out of brake to what is question...previous mode I take it...

Reply:
changed to BRAKE mode